service/worker/scheduler/testdata/generate_history.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -x
+
+id=sched1
+
+trap "temporal schedule delete -s $id" EXIT
+
+temporal schedule create -s $id \
+  --overlap-policy bufferall \
+  --interval 10s \
+  --jitter 8s \
+  -w mywf \
```

Comment:
Is this something that needs to be registered before this script is run? At a higher level, I think we should document this script better. What does it do, why would you use it, how do you run it, should we rerun it for newer versions later?

Reply:
nah, it's just a dummy name. the workflow will time out in 5s since there are no workers.
added some comments on top.